Cdnfsd_TransparentCDN_Engine.php

```diff
@@ -0,0 +1,233 @@
+<?php
+namespace W3TC;
+
+/**
+ * 
+ */
+
+if (!defined('W3TC_CDN_TRANSPARENTCDN_PURGE_URL')) define('W3TC_CDN_TRANSPARENTCDN_PURGE_URL', 'https://api.transparentcdn.com/v1/companies/%s/invalidate/');
+if (!defined('W3TC_CDN_TRANSPARENTCDN_AUTHORIZATION_URL')) define('W3TC_CDN_TRANSPARENTCDN_AUTHORIZATION_URL', ' https://api.transparentcdn.com/v1/oauth2/access_token/');
+
+class Cdn_TransparentCDN_Api {
+	var $_token;
+	var $_config;
+    /**
+     * PHP5 Constructor
+     *
+     * @param array $config
+     */
+    function __construct($config = array()) {
```

Comment:
Add a space after opening parenthesis and before the closing.
`function __construct( $config = array() ) {`